src/release/options.d

```diff
@@ -52,8 +55,11 @@ Options parseOpts ( string[] opts )
     auto help_info = getopt(opts,
            "log|l", format("Set the logging level, one of %s",
                            iota(LogLevel.min, LogLevel.max).map!LogLevel), &options.logging,
-           "verbose", "Set logging to verbose", &verbose);
+           "verbose", "Set logging to verbose", &verbose,
 
+           "release_subsequent|f",
```

Comment:
Sorry for the ignorance, but will this translate to `--release-subsequent` or will it be `--release_subsequent`? If so please please make it `--release-subsequent` :)

Also, I guess that `f` is a short option (`-f`), why `f`? That usually means `--force` and there is no `f` at all in the whole `release_subsequent`...

Also, maybe `--release-all` / `-a`?